Loadsys/Sniffs/NamingConventions/ValidPrivateProtectedFunctionNameSniff.php

```diff
@@ -98,7 +98,7 @@ protected function processTokenWithinScope(PHP_CodeSniffer_File $phpcsFile, $sta
 		if ($isPublic === true) {
 			if ($methodName[0] === '_') {
 				$error = 'Public method name "%s" must not be prefixed with underscore';
-				$phpcsFile->addError($error, $stackPtr, 'PublicWithUnderscore', $errorData);
+				$phpcsFile->addWarning($error, $stackPtr, 'PublicWithUnderscore', $errorData);
 				return;
```

Comment:
I should note that in this case, we wouldn't want to set the severity all the way down to zero, but just bump it down from the default "error" range into the "warning" range. That way we'll still _see_ them when running locally as intended, but can still easily suppress it during Travis builds with `-n`.